lib/models/device.coffee

```diff
@@ -1560,12 +1583,16 @@ getDeviceModel = (deps, opts) ->
 	# });
 	###
 	exports.disableTcpPing = (uuidOrId, callback) ->
-		exports.get(uuidOrId).then (device) ->
+		exports.get uuidOrId,
+			select: 'id'
+			expand: belongs_to__application: $select: 'id'
+		.then (device) ->
 			return request.send
-				method: 'DELETE'
+				method: 'POST'
```

Comment:
I guess that this changed at some point

Reply:
As far as I can tell, it's needed this [since basically forever](https://github.com/resin-io/resin-api/commit/2e7c2896c82fbc530172c27b3774e9c501a70fa2), so it's never worked... The API supervisor proxy requires a POST, plus the real method to use in the body.

Horia fixed the same issue in `ping` recently. The problem is we just don't have automated tests for this stuff right now, so any useful but rarely used methods like this that require real devices can be silently broken :cry: